src/local/cost_matrix.rs

```diff
@@ -172,28 +162,24 @@ impl SparseCostMatrix {
         SparseCostMatrix { inner: HashMap::new() }
     }
 
-    pub fn get(&self, x: u8, y: u8) -> u8 {
-        assert!(x < 50, "out of bounds x: {}", x);
-        assert!(y < 50, "out of bounds y: {}", y);
-        if let Some(ref_val) = self.inner.get(&(x, y)) {
+    pub fn get(&self, xy: RoomXY) -> u8 {
+        if let Some(ref_val) = self.inner.get(&xy) {
```

Comment:
This can also be done as: ``
self.inner.get(&gy).as_deref().unwrap_or(0)
``

Reply:
I can't believe I missed that, makes some RefCell-like structs I was writing much cleaner too.